ingestion_server/ingestion_server/elasticsearch_models.py

```diff
@@ -330,12 +336,22 @@ def database_row_to_elasticsearch_doc(row, schema):
             category=row[schema["category"]],
             duration=row[schema["duration"]],
             length=length,
+            filetype=filetype,
```

Comment:
We should probably only have the `extension` here because it is used in search as a filter (`extension=mp3`). When querying, we will only be using the `extension` field, and the `filetype` should already be returned as part of the `source` object when #684 is merged.

Reply:
It is necessary to have it here now for having the `filetype` appearing in the search results instead of always `null`, it can be changed later in that PR if found unnecessary. Please follow [this comment](https://github.com/WordPress/openverse-api/pull/769#discussion_r906151591) about the filetype/extension discussion.